daphne/src/bin/dapf.rs

```diff
@@ -125,7 +125,7 @@ fn main() -> Result<()> {
                 reqwest::header::HeaderValue::from_static(constants::MEDIA_TYPE_REPORT),
             );
             let resp = http_client
-                .post(Url::parse(leader_url)?.join("/upload")?)
+                .post(Url::parse(leader_url)?.join("upload")?)
```

Comment:
Does this insert the path separator `/`?

Reply:
@nakatsuka-y observed an interesting and surprising behavior of the `url` crate. If `leader_url` ends ends with `/thing/`, then joining with `/upload` will overwrite `/thing`. For example if the the url is `example.com/thing/`, then the new url would be `example.com/upload`. Removing the `/` prefix ensures we get what we want, which is `example.com/thing/upload`. Very strange.